agent/config/config.go

```diff
@@ -429,6 +432,17 @@ func getDockerStopTimeout() time.Duration {
 	return dockerStopTimeout
 }
 
+func getContainerStartTimeout() time.Duration {
+	var containerStartTimeout time.Duration
+	parsedStartTimeout := parseEnvVariableDuration("ECS_CONTAINER_START_TIMEOUT")
+	if parsedStartTimeout >= minimumContainerStartTimeout {
+		containerStartTimeout = parsedStartTimeout
+	} else if parsedStartTimeout != 0 {
```

Comment:
The `parsedStartTimeout` is initialized by calling `parseEnvVariableDuration`. `parseEnvVariableDuration` will parse the `ECS_CONTAINER_START_TIMEOUT`, if `ECS_CONTAINER_START_TIMEOUT` is in the correct format, then the function will return the parsed time duration, otherwise it will return a time duration of 0s, and log a warning at the same time. So if `parsedStartTimeout` is 0s (note that `parsedStartTimeout == 0` will return true if `parsedStartTimeout` is 0s), then we don't need to log it again.

Reply:
I still don't quite understand what's happening here. Might be simpler to do something like this:

```golang
if parsedStartTimeout < minimumContainerStartTimeout {
 log.Infof(...) 
 parsedStartTimeout = parsedStartTimeout
}

return containerStartTimeout
```

Reply:
Things will go wrong when `ECS_CONTAINER_START_TIMEOUT` is set to be 0s, I will delete the `parsedStartTimeout != 0`, thanks!